.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ name: build
 on: [push, pull_request]
 
 env:
-  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx2g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
```

Comment:
I’m hoping that provisioning less memory to the JVM will leave more available to Konan.